wheatley/rhythm.py

```diff
@@ -127,6 +155,12 @@ def expect_bell(self, expected_bell, row_number, place, expected_stroke):
         if expected_bell not in self._early_bells[expected_stroke]:
             self._expected_bells[expected_stroke].add(expected_bell)
 
+    def change_setting(self, key, value):
+        self._inner_rhythm.change_setting(key, value)
+
+        if key == "sensitivity":
+            self.logger.warning(f"NOT IMPLEMENTED: setting sensitivity to {value}.")
```

Comment:
I don't think this needs to be here, it will be logged  (or handled in the future) by the inner_rhythm

Reply:
I guess.  The thing is that I expect `WaitForUserRhythm` to use the sensitivity control, and so the 'not implemented' log is still mostly valid.  But I suppose the import thing is that we get some logging, so I'll fix this...

Reply:
I now have an amusing image of `WaitForUserRhythm` using sensitivity to decide whether to politely hold right up over someone who is slow or to aggressively clip them ...